zerver/lib/send_email.py

```diff
@@ -36,12 +36,16 @@ def build_email(template_prefix: str, to_user_id: Optional[int]=None,
         # Change to formataddr((to_user.full_name, to_user.email)) once
         # https://github.com/zulip/zulip/issues/4676 is resolved
         to_email = to_user.email
+        context.update({
+            'realm_name_in_notifications': to_user.enable_realm_name_in_notifications
```

Comment:
Let's just set this in the missed message pathway, to keep things simple. Probably do_send_missedmessage_events_reply_in_zulip?

Reply:
OK, set this into `do_send_missedmessage_events_reply_in_zulip`